### ISSUE TITLE
[Feature request] Give buttons a color for easy learning 

### ISSUE THREAD

**Issue by grammoboy2** [milkytracker/MilkyTracker#227]
For children and educational purposes, it would be nice if the buttons would have a color. 

Click the 'green button' 
Click the 'red button' 

So children can use milkytracker more easy

**Comment by 44100hertz** [comment]
If this feature is rejected, I may try to create it myself as an exercise.

**Comment by TheAwesome98-Real** [comment]
I agree the play should be green, stop red... but it’s not hard to understand "click the button whose label is Smp. Ed" (unless the kid is like 5 then I understand)
